source/gui/widgets/skeletons/text_editor.cpp

```diff
@@ -1456,6 +1456,10 @@ namespace nana{	namespace widgets
 					select_.ignore_press = (arg_focus::reason::mouse_press == arg.focus_reason);
 				}
 			}
+
+			// Reset the selection as it is sticky in the "double click" path.
+			if (!arg.getting)
+				select(false);
```

Comment:
Very nice tip! Let's keep the selection even though it loses focus. The issue where multiple textboxes on a single form would be marked as a known issue and will be fixed in 1.6.
The new fix will be like this. When a text is selected, deselect selection in other textbox.